hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/exec/scope/HandleHederaOperations.java

```diff
@@ -140,7 +140,7 @@ public long contractCreationLimit() {
     @Override
     public long lazyCreationCostInGas() {
         // TODO - implement correctly
-        return 1L;
+        return 500_000L;
```

Comment:
@tinker-michaelj @Neeharika-Sompalli 

I couldn't find the lazy creation gas cost calculation in the mono repo? Is it a config value, and could you point me to where the calculation/config is?